crates/aptos/e2e/local_testnet.py

```diff
@@ -8,7 +8,8 @@
 import time
 
 import requests
-from common import FAUCET_PORT, METRICS_PORT, NODE_PORT, Network, build_image_name
+from common import (FAUCET_PORT, METRICS_PORT, NODE_PORT, Network,
```

Comment:
Please run isort first and then black.

Reply:
Hmm I guess order matters. I just added a simple script to run linter related for e2e. 

https://github.com/aptos-labs/aptos-core/pull/8838